libs/dal/src/repositories/user/user.entity.ts

```diff
@@ -47,4 +47,6 @@ export class UserEntity {
     times: number;
     lastFailedAttempt: string;
   };
+
+  userHashForIntercom?: string;
```

Comment:
thinking out loud because of: https://linear.app/novu/issue/NV-1456/enable-hmac-encryption-on-our-clients-notification-center

Should we changes this to:

```
servicesHashes: { intercom: string }
```
to support other service in the future including "Novu" ?

Reply:
Yes this option will be better to accomodate future requirements

Reply:
@jainpawan21 do you want to tackle this already in this PR?

Reply:
Yes, I will update this PR
do we need to run any migration to add extra property in user schema?

Reply:
No, mongo db  doesn't need that :)